project-base/storefront/tailwind.config.js

```diff
@@ -14,36 +14,28 @@ module.exports = {
             xxl: em(1560),
```

Comment:
Cookie consent banner UI is broken
<img width="1047" alt="image" src="https://github.com/shopsys/shopsys/assets/55181354/375d3402-c231-433d-92d7-cf915c4ecbfc">